### ISSUE TITLE
Segments from route.txt do not match map layer

### ISSUE THREAD

**Issue by TomWaks** [GraphAlgoX/GraphMM-Master#3]
In order to visualize the sample data, I selected the first trajectory from the file trace.txt (#,6221427912124254782,6221427912124254782) and placed markers on the map. I also added segments (segmentId from trace.txt for each trajectory point) along with the coordinates of the segment from the file road.txt. Here is the result.
![image](https://github.com/user-attachments/assets/ea5d471d-7cd8-41c6-9f28-b23fa6f5f41c)

How to explain my result? Drawn segments do not match map layer. Why?

When trying to visualize the entire road network, the effect is the same.

**Comment by wlc185** [comment]
Yes, I noticed that too. I've tried printing the entire network, and it does look quite strange.
![0H7PJO() `~Y RN$GLV()R](https://github.com/user-attachments/assets/6679b291-5bf2-4b90-a3ac-6a9feb9202e6)

**Comment by Yeah2333** [comment]
@TomWaks @wlc185 Hi, I also noticed this issue. Have you found out the reason?

![Image](https://github.com/user-attachments/assets/af6029fb-3e81-45df-8dd0-7037c5cb4088)

**Comment by wlc185** [comment]
不知道，同学，我看可能是他原代码（build_road_graph.py）对路网进行了转换，就是将路段转换成网格形式，可能作者上传的就是转换后的，但是我直接跑他给的数据效果也很差，200个epoch，精度只有20%左右，跑自己的数据精度直接为0，现在还在查是什么原因，同学你啥进度呀，你有跑自己的数据集吗，效果咋样

![Image](https://github.com/user-attachments/assets/efd11d3e-01f8-4c10-bd44-685ad55a10cc)
